src/LogDocumentLinkProvider.ts

```diff
@@ -78,19 +81,29 @@ export class LogDocumentLinkProvider implements vscode.DocumentLinkProvider {
         return this.fileMaps[pkgPath];
     }
 
-    public addCustomLink(customLink: CustomDocumentLink) {
-        let match: RegExpExecArray;
-
-        let fileMap = this.getFileMap(customLink.pkgPath);
-        if (fileMap) {
-            let uri = vscode.Uri.file(fileMap.src);
+    public addCustomLink(customLink: CustomDocumentLink, isFilePath: boolean = false) {
```

Comment:
This function doesn't share any code whatsoever between isFilePath: true | false, ...so how about we break these into two functions:
 - `addCustomPkgLink`
 - `addCustomFileLink`